nvm/tasks/per_user.yml

```diff
@@ -0,0 +1,44 @@
+---
+- become_user: '{{ nvm_user_config.user }}'
+  become: true
+  block:
```

Comment:
I can see the block below being conditional, but can't see why you have wrapped the rest in a block. Could you not have put the become_user and become into the loop task in main.yml above and put these as individual tasks? Is this just a style thing again?

Reply:
Become doesn't pass through when doing an include, so it has to be on a block like this. I think there may be a fix in the works to allow it to apply to the include.